web/src/lib/domain-logic/getNavBarBusinessTitle.ts

```diff
@@ -26,6 +34,15 @@ export const getNavBarBusinessTitle = (
     }
   };
 
+  if (foreignBusinessType === "NEXUS") {
+    if (needsNexusDbaName && nexusDbaName === "") {
+      return Config.navigationDefaults.navBarUnnamedDbaBusinessText;
+    }
+    if (needsNexusDbaName && nexusDbaName !== "") {
+      return nexusDbaName;
+    }
+  }
```

Comment:
[dust] You shouldn't actually need the check `foreignBusinessType === "NEXUS"`. `needsNexusDbaName` being `true` implies that this is a Nexus business. You could restructure this to be:
```
if (needsNexusDbaName) {
  if ( nexusDbaName === "") {
  } else {
  }
}
```

Reply:
This is completely unrelated to the work you're doing with this PR.

Now that name availability is data stored and associated with a business object, `needsNexusDbaName` is somewhat redundant. I would make an argument that this information can be inferred by other data and keeping both increases the likelihood of inconsistencies in our data.

Reply:
Sure but isn't that more just something to take into account for a refactor that eliminates the needsNexusDbaName object?

If not do you have specific fields or structure for how you would want to infer that?

Reply:
Yeah, the 2nd part is just me thinking out loud via the comments section.

Like I said, we track business name availability. If the business name is unavailable, and you are a foreign business, then you would need a dba name. Because we can calculate this from these other pieces of information, I would argue it shouldn't be tracked as part of the object.

Reply:
Gotcha, I'll write a follow up refactor for the notion listed above